Drop Ruby < 2.5 support

tagliala opened the pull request:
Instead of going for 2.4, which is the minimum required Ruby version by
simplecov 0.18, it has been decided to go for 2.5 and newer

Ref: #5

t-mario-y commented on simplecov_lcov_formatter.gemspec:
```diff
@@ -13,6 +13,8 @@ Gem::Specification.new do |s|
   s.homepage = 'https://github.com/t-mario-y/simplecov_lcov_formatter'
   s.licenses = 'MIT'
 
+  s.required_ruby_version = '>= 2.4'
```
We need to think 'how old ruby version to support?'

- Describing myself, I'm using Ruby for just web app development and not familier with old versions, especially before 2.6. 
- On GitHub Actions https://github.com/ruby/setup-ruby can use very old version, but older language version will make difficult to inspect edge cases on developing machine.
- SimpleCov tried to drop 2.5 & 2.6, but reverted on  https://github.com/simplecov-ruby/simplecov/pull/1108 (This had no strong reason, just 'as it is' I understood)
- Your coveralls ruby reborn https://github.com/tagliala/coveralls-ruby-reborn/blob/525f89c1825921bacc6a2a636082a5f504947243/coveralls-ruby.gemspec#L27 will support 2.6 and above

So, currently '>= 2.5' is enough for using with SimpleCov, coveralls.

Please tell me if other points to consider is remaining.

tagliala replied:
Not a problem at all, I will bump to 2.5

t-mario-y replied:
I understood, at SimpleCov 0.18.0 it does support Ruby 2.4 https://github.com/simplecov-ruby/simplecov/blob/v0.18.0/simplecov.gemspec#L24
and soon dropped next 0.19.0 https://github.com/simplecov-ruby/simplecov/releases/tag/v0.19.0

But conclusion remains, just 2.5 and above seems enough I think.

tagliala committed:
Drop Ruby < 2.5 support

Instead of going for 2.4, which is the minimum required Ruby version by
simplecov 0.18, it has been decided to go for 2.5 and newer

Ref: #5

t-mario-y reviewed (approved):
LGTM, thank you so much!
